code/game/objects/items/weapons/implants/implant_explosive.dm

```diff
@@ -45,15 +45,16 @@
 		return
 	timed_explosion()
 
-/obj/item/weapon/implant/explosive/implant(mob/source)
-	var/obj/item/weapon/implant/explosive/imp_e = locate(src.type) in source
```

Comment:
for all these implants we no longer need to check everything in the mob's content.